### PR TITLE
Templets added.

### PR DESCRIPTION
**Pull request by KarandeepSinghCoder**
C

### CODE REVIEW

File: .github/contributing-guidelines-practices.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+**Feature Requests and Bug Reports**
+* When you file a feature request or when you are submitting a bug report to the [issue tracker](https://github.com/fossasia/susi_server/issues), make sure you add steps to reproduce it. Especially if that bug is some weird/rare one.
+
+**Join the development**
+* Before you join development, please set up the project on your local machine, run it and go through the application completely. Press on any button you can find and see where it leads to. Explore. (Don't worry ... Nothing will happen to the app or to you due to the exploring :wink: Only thing that will happen is, you'll be more familiar with what is where and might even get some cool ideas on how to improve various aspects of the app.)
+* If you would like to work on an issue, drop in a comment at the issue. If it is already assigned to someone, but there is no sign of any work being done, please free to drop in a comment so that the issue can be assigned to you if the previous assignee has dropped it entirely.
+
+Do read the [Open Source Developer Guide and Best Practices at FOSSASIA](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
```
Comment (YashKumarVerma) [inline .github/contributing-guidelines-practices.md:20]: FOSSASIA ?

File: .github/contributing-guidelines-practices.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+* When you make very very minor changes to a PR of yours (like for example fixing a failing travis build or some small style corrections or minor changes requested by reviewers) make sure you squash your commits afterwards so that you don't have an absurd number of commits for a very small fix. (Learn how to squash at https://davidwalsh.name/squash-commits-git )
+* When you're submitting a PR for a UI-related issue, it would be really awesome if you add a screenshot of your change or a link to a deployment where it can be tested out along with your PR. It makes it very easy for the reviewers and you'll also get reviews quicker.
+
+**Code Styleguide**
+* Do follow the .editorconfig file regarding maintaining of code style (It's mandatory).
+* For more information regarding .editorconfig file, see [editorconfig](http://editorconfig.org/#download)
+
+**Feature Requests and Bug Reports**
+* When you file a feature request or when you are submitting a bug report to the [issue tracker](https://github.com/fossasia/susi_server/issues), make sure you add steps to reproduce it. Especially if that bug is some weird/rare one.
```
Comment (YashKumarVerma) [inline .github/contributing-guidelines-practices.md:14]: SUSI Server ?

**Comment by YashKumarVerma** [comment]
Use the request review option to get it merged when the changes are done.

File: .github/pr-template.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+Fixes #[Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issues #IssueNumber]
+
+Changes: [Add here what changes were made in this issue and if possible provide links.]
+
+Screenshots for the change:
```
Comment (YashKumarVerma) [inline .github/pr-template.md:5]: rename it to `.github/PULL_REQUEST_TEMPLATE.md`

Comment (YashKumarVerma) [reply .github/pr-template.md:5]: .github is the folder

Comment (YashKumarVerma) [reply .github/pr-template.md:5]: @KarandeepSinghCoder

Comment (KarandeepSinghCoder) [reply .github/pr-template.md:5]: I am trying to fix it . Thanks for the highlighted Issue

**Comment by YashKumarVerma** [comment]
Thanks a lot for your contributions.

**Pull request merged**